src/api/userChallenge/controller.ts

```diff
@@ -110,16 +111,30 @@ route.get('/', async (req: Request, res: Response, next: NextFunction) => {
     return next(new UnauthorizedError(ErrorMessages.unauthorized));
   }
 
+  const status = req.query.status as ChallengeStatus;
```

Comment:
I have added the validation lorgic here.

We always should work with safe/parsed values because we don`t know what we can get